### PR TITLE
v5.17.21

### PR DESCRIPTION
**Pull request by MBilalShafi**
### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```sh
git push upstream master:docs-v5 -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v5)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-v5` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,39 @@
...
+
+### `@mui/x-date-pickers@v5.0.16` / `@mui/x-date-pickers-pro@v5.0.16`
+
+#### Changes
+
+- [pickers] Add missing components to `themeAugmentation` (#7677) @LukasTy
+- [l10n] Add Czech (cs-CZ) locale (#7666) @LukasTy
+- [l10n] Add Russian (ru-RU) locale (#7708) @LukasTy
+- [l10n] Improve Spanish (es-ES) locale (#7614) @WiXSL
+
```
Comment (LukasTy) [inline CHANGELOG.md:38]: I've just merged https://github.com/mui/mui-x/pull/7723
Could we also include it in the changelog?
```suggestion

### Docs

- [docs] Add info callout about available component `slots` (#7723) @Vivek-Prajapatii
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,39 @@
...
+- [l10n] Add missing core locales for `MuiTablePagination` (#7719) @MBilalShafi
+- [l10n] Improve Japanese (ja-JP) locale (#7627) @makoto14
+- [l10n] Improve Vietnamese (vi-VN) locale (#7601) @SpacerZ
+
+### `@mui/x-date-pickers@v5.0.16` / `@mui/x-date-pickers-pro@v5.0.16`
+
+#### Changes
+
+- [pickers] Add missing components to `themeAugmentation` (#7677) @LukasTy
+- [l10n] Add Czech (cs-CZ) locale (#7666) @LukasTy
```
Comment (LukasTy) [inline CHANGELOG.md:35]:
```suggestion
- [l10n] Add Czech (cs-CZ) locale (#7666) @OndrejHj04
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,39 @@
...
+- [l10n] Add Russian (ru-RU) locale (#7708) @LukasTy
```
Comment (LukasTy) [inline CHANGELOG.md:36]:
```suggestion
- [l10n] Add Russian (ru-RU) locale (#7708) @rstmzh
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,39 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.17.21
+
+_Jan 27, 2023_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Belarusian (be-BY) and improve some locales
```
Comment (LukasTy) [inline CHANGELOG.md:12]:
```suggestion
- 🌍 Add Belarusian (be-BY), Czech (cs-CZ), and Russian (ru-RU) locales
- 🌍 Improve Belarusian (be-BY), Slovak (sk-SK), Japanese (ja-JP), Vietnamese (vi-VN), and Spanish (es-ES) locales
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,39 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.17.21
+
+_Jan 27, 2023_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
```
Comment (LukasTy) [inline CHANGELOG.md:10]:
```suggestion
We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,44 @@
...
+
+- 🌍 Add Belarusian (be-BY), Czech (cs-CZ), and Russian (ru-RU) locales
+- 🌍 Improve Belarusian (be-BY), Slovak (sk-SK), Japanese (ja-JP), Vietnamese (vi-VN), and Spanish (es-ES) locales
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.17.21` / `@mui/x-data-grid-pro@v5.17.21` / `@mui/x-data-grid-premium@v5.17.21`
+
+#### Changes
+
+- [DataGrid] Add `title` to cells (#7695) @thupi
```
Comment (MBilalShafi) [inline CHANGELOG.md:20]:
```suggestion
- [DataGrid] Add `title` attribute to cells (#7695) @thupi
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,44 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.17.21
+
+_Jan 27, 2023_
+
+We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Belarusian (be-BY), Czech (cs-CZ), and Russian (ru-RU) locales
+- 🌍 Improve Belarusian (be-BY), Slovak (sk-SK), Japanese (ja-JP), Vietnamese (vi-VN), and Spanish (es-ES) locales
```
Comment (MBilalShafi) [inline CHANGELOG.md:13]:
```suggestion
- 🌍 Improve Slovak (sk-SK), Japanese (ja-JP), Vietnamese (vi-VN), and Spanish (es-ES) locales
```

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,39 @@
...
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Belarusian (be-BY) and improve some locales
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.17.21` / `@mui/x-data-grid-pro@v5.17.21` / `@mui/x-data-grid-premium@v5.17.21`
+
+#### Changes
+
+- [DataGrid] Add `title` to cells (#7695) @thupi
```
[the remark this answers is not in the document]
Comment (MBilalShafi) [reply CHANGELOG.md]: You had the page not refreshed may be 🙂

**Pull request merged**